src/main/java/baseball/UserInput.java

```diff
@@ -0,0 +1,37 @@
+
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserInput {
+	
+	private List<Integer> user = new ArrayList<Integer>();
```

Comment:
```suggestion
	private List<Integer> user = new ArrayList<>();
```
다음과 같이 오른쪽의 제네릭은 생략하실 수 있을 것 같아요